.github/workflows/compute-impacted-libraries.yml

```diff
@@ -0,0 +1,63 @@
+name: "Compute libraries impacted by the change"
+
+on:
+  workflow_call:
+    outputs:
+      impacted_libraries:
+        description: "JSON list of libraries impacted by the change"
+        value: ${{ jobs.main.outputs.impacted_libraries }}
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    outputs:
+      impacted_libraries: ${{ steps.compute_impacted_libraries.outputs.result }}
+    steps:
+      - uses: actions/checkout@v4
+      - run:  |
+          git fetch origin ${{ github.event.pull_request.base.sha || github.sha }}
+          git diff --name-only HEAD ${{ github.event.pull_request.base.sha || github.sha }} >> modified_files.txt
```

Comment:
is this going to work for merge actions? I mean when we merge something to the main branch.
Anyway, Totally agree for the PRs, but I advocate for run all cases for all libraries when we merge to the main branch

Reply:
indeed, fixing that point

Reply:
Done in [44a4791](https://github.com/DataDog/system-tests/pull/2356/commits/44a4791e4b0b918bd6ce0b10f4ba6846deed65ea)